backend/src/main/resources/db/migration/V2__Populate_questions.sql

```diff
@@ -0,0 +1,12 @@
+-- initial question data
+insert into questions (id, placeholder, cs, eng)
```

Comment:
to je spíš `name` toho polička než `placeholder`, ale to je detail

Reply:
beru na vedomi, zatim necham placeholder, kdyztak to prejmenujeme v budoucnu 👍